src/pip/_internal/cmdoptions.py

```diff
@@ -434,7 +434,10 @@ def only_binary():
     '-b', '--build', '--build-dir', '--build-directory',
     dest='build_dir',
     metavar='dir',
-    help='Directory to unpack packages into and build in.'
+    help='Directory to unpack packages into and build in. '
+         'Initial build would still take place in direcotry set by TMPDIR '
+         'environment variable, (TEMP on windows) To change this behavior,'
```

Comment:
- an extra comma here after `variable`
- `To` is capitalised in the middle of the sentence
- `windows` should be `Windows`
- there is no space after the last comma